scripts/process_data.py

```diff
@@ -944,28 +1149,28 @@ def main(self) -> None:
         summary_log.append(downscale_images(image_dir, self.num_downscales, verbose=self.verbose))
 
         # Run Colmap
-        colmap_dir = self.output_dir / "colmap"
+        opensfm_dir = self.output_dir / "opensfm"
         if not self.skip_colmap:
-            colmap_dir.mkdir(parents=True, exist_ok=True)
+            opensfm_dir.mkdir(parents=True, exist_ok=True)
 
-            run_colmap(
+            run_opensfm(
                 image_dir=image_dir,
-                colmap_dir=colmap_dir,
-                camera_model=CAMERA_MODELS["fisheye"],
+                opensfm_dir=opensfm_dir,
+                camera_model=CAMERA_MODELS["equirectangular"],
                 gpu=self.gpu,
                 verbose=self.verbose,
                 matching_method=self.matching_method,
-                colmap_cmd=self.colmap_cmd,
+                opensfm_install=Path("/scratch2/ksalahi/OpenSfM"),
             )
 
         # Save transforms.json
-        if (colmap_dir / "sparse" / "0" / "cameras.bin").exists():
+        if (opensfm_dir / "sparse" / "0" / "cameras.bin").exists():
             with CONSOLE.status("[bold yellow]Saving results to transforms.json", spinner="balloon"):
-                num_matched_frames = colmap_to_json(
-                    cameras_path=colmap_dir / "sparse" / "0" / "cameras.bin",
-                    images_path=colmap_dir / "sparse" / "0" / "images.bin",
+                num_matched_frames = opensfm_to_json(
+                    cameras_path=opensfm_dir / "sparse" / "0" / "cameras.bin",
+                    images_path=opensfm_dir / "sparse" / "0" / "images.bin",
                     output_dir=self.output_dir,
-                    camera_model=CAMERA_MODELS["fisheye"],
+                    camera_model=CAMERA_MODELS["equirectangular"],
```

Comment:
@kerrj Should we revert to the original for now? Or should we be using equirect images as the de-facto standard for insta360 from here on?

Reply:
Also, the `install_checks.check_colmap_installed()` checks are redundant if we are using OpenSfM

Reply:
Let's keep the standard methods for now. We can change the standard in a different PR.